libs/api_common/windows_helpers.cpp

```diff
@@ -86,55 +84,23 @@ ebpf_result_t
 get_program_type_info(const ebpf_program_info_t** info)
 {
     const GUID* program_type = reinterpret_cast<const GUID*>(global_program_info.type.platform_specific_data);
-    ebpf_result_t result;
+    ebpf_result_t result = EBPF_SUCCESS;
     ebpf_program_info_t* program_info;
-    const uint8_t* encoded_data = nullptr;
-    size_t encoded_data_size = 0;
-    bool fall_back = false;
 
     // See if we already have the program info cached.
     auto it = _program_info_cache.find(*program_type);
     if (it == _program_info_cache.end()) {
         // Try to query the info from the execution context.
         result = get_program_info_data(*program_type, &program_info);
-        if (result != EBPF_SUCCESS) {
-            fall_back = true;
-        } else {
-            _program_info_cache[*program_type] = ebpf_program_info_ptr_t(program_info);
-        }
-    }
-
-    if (fall_back) {
-        // Fall back to using static data so that verification can be tried
-        // (e.g., from a netsh command) even if the execution context isn't running.
```

Comment:
Have you verified that verification still succeeds in netsh even if the execution context isn't running?  (We shouldn't regress this right now.)  I thought this wouldn't work until after issue #223 is done in which case this would be blocked on doing that first.

Reply:
Yes I have verified netsh ebpf show verification works. #223 would need changes in libs\api_common\windows_platform_common.cpp not this file.

Reply:
@shankarseal I just verified that show verification fails, and it appears to be a regression caused by this change.   How did you verify it works and what path is taken to let it work?

Reply:
#223 will not fix this. I think if static data is removed, verification cannot work without execution context (and the extension drivers) loaded.